### PR TITLE
Bulk app translations: special chars in simple nodes

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?174470

Could instead use an HTML parser to unescape simple nodes, but it makes for cleaner code to get rid of the simple/complex distinction.

@emord 

code buddy @sravfeyn

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 1]
```diff
@@ -151,9 +151,9 @@ class BulkAppTranslationBasicTest(BulkAppTranslationTestBase):
         ("module1_form1", (
           ("question1-label", "in english", "in french", "", "", "", "", "", ""),
           ("question2-label", "one &lt; two", "un &lt; deux", "", "", "", "", "", ""),
-          ("question2-item1-label", "item1", "item1", "", "", "", "", "", ""),
+          ("question2-item1-label", "item1's label", "item1's label", "", "", "", "", "", ""),
```
Comment (emord) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: Could you add a test for this change?

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 2]
```diff
@@ -219,8 +219,10 @@ def test_change_upload(self):
         )
 
         # Test special characters and output refs
-        self.assert_question_label("one &lt; two", 0, 0, "en", "/data/question2")
-        self.assert_question_label("un &lt; deux", 0, 0, "fra", "/data/question2")
+        #jls
+        self.assert_question_label("one < two", 0, 0, "en", "/data/question2")
+        self.assert_question_label("un < deux", 0, 0, "fra", "/data/question2")
+        self.assert_question_label("question3's label", 0, 0, "fra", "/data/question3")
```
Comment (emord) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: Possibly add a test for something like `blah < blah <output value='blah' /> <blah />` ?

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 3]
```diff
@@ -149,14 +149,14 @@ class BulkAppTranslationBasicTest(BulkAppTranslationTestBase):
           ("baz (ID Mapping Value)", "detail", "quz", ""),
         )),
         ("module1_form1", (
-          ("question1-label", "in english", "in french", "", "", "", "", "", ""),
+          ("question1-label", "in english", "it's in french", "", "", "", "", "", ""),
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: E121 continuation line under-indented for hanging indent

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 4]
```diff
@@ -149,14 +149,14 @@ class BulkAppTranslationBasicTest(BulkAppTranslationTestBase):
...
           ("question2-label", "one &lt; two", "un &lt; deux", "", "", "", "", "", ""),
           ("question2-item1-label", "item1", "item1", "", "", "", "", "", ""),
           ("question2-item2-label", "item2", "item2", "", "", "", "", "", ""),
-          ("question3-label", "question3", "question3", "", "", "", "", "", ""),
+          ("question3-label", "question3", "question3&#39;s label", "", "", "", "", "", ""),
           ("question3/question4-label", 'question6: <output value="/data/question6"/>', 'question6: <output value="/data/question6"/>', "", "", "", "", "", ""),
           ("question3/question5-label", "English Label", "English Label", "", "", "", "", "", ""),
-          ("question7-label", "question7", "question7", "", "", "", "", "", "")
+          ("question7-label", 'question1: <output value="/data/question1"/> &lt; 5', "question7", "", "", "", "", "", "")
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: E501 line too long (121 > 115 characters)

**Comment by orangejenny** [comment]
@emord updated tests

ignoring dimagimon

**Pull request merged**